third-party/massivethreads/Makefile

```diff
@@ -5,6 +5,19 @@ endif
 CHPL_MAKE_HOST_TARGET = --target
 include $(CHPL_MAKE_HOME)/make/Makefile.base
 
+#
+# Cray X* builds are cross-compilations.
+#
+ifneq (, $(filter cray-x%,$(CHPL_MAKE_TARGET_PLATFORM)))
+CHPL_MASSIVETHREADS_CFG_OPTIONS += --host=x86_64-cle-linux-gnu
+endif
+
+CHPL_MASSIVETHREADS_CFG_OPTIONS += --enable-malloc-wrapper=no \
+                                   --enable-eco-mode=no \
+                                   --enable-bind-workers=no
+
+CHPL_MASSIVETHREADS_CFG_OPTIONS += $(CHPL_MASSIVETHREADS_MORE_CFG_OPTIONS)
```

Comment:
I don't see CHPL_MASSIVETHREADS_MORE_CFG_OPTIONS defined anywhere. Is that on purpose?